metalearn/metafeatures/metafeatures.py

```diff
@@ -59,7 +58,7 @@ class Metafeatures(object):
     IDS: List[str] = [mf_id for mfs_info in _mfs_info for mf_id in mfs_info.keys()]
 
     @classmethod
-    def list_metafeatures(cls, group="all"):
+    def list_metafeatures(cls, group: consts.MetafeatureGroup='all'):
```

Comment:
`consts.MetafeatureGroup` is an Enum, but the default value is a string. Since we have the enums, let's use them as much as possible, instead of the string literal. @JeremyRees can you make this change throughout the code?